frontend/src/components/landing/PlanGrid.tsx

```diff
@@ -0,0 +1,530 @@
+                : l10n.getString("plan-grid-card-btn")}
+            </LinkButton>
+          </dd>
+        </dl>
+      </section>
+    </div>
+  );
+};
+
+type PricingToggleProps = {
+  yearlyBilled: {
+    monthly_price: string;
+    yearly_price: string;
+    subscribeLink: string;
+    gaViewPing: Parameters<typeof useGaViewPing>[0];
+    plan: Plan;
+  };
+  monthlyBilled: {
+    monthly_price: string;
+    subscribeLink: string;
+    gaViewPing: Parameters<typeof useGaViewPing>[0];
+    plan: Plan;
+  };
+};
+const PricingToggle = (props: PricingToggleProps) => {
+  const l10n = useL10n();
+  const gaEvent = useGaEvent();
+  const yearlyButtonRef = useGaViewPing(props.yearlyBilled.gaViewPing);
+  const monthlyButtonRef = useGaViewPing(props.monthlyBilled.gaViewPing);
+
+  return (
+    <PricingTabs defaultSelectedKey="yearly">
+      <Item
+        key="yearly"
+        title={l10n.getString("plan-matrix-price-period-yearly")}
+      >
+        <div className={styles["price-text"]}>
+          <small>
+            {l10n.getString("plan-matrix-price-yearly-calculated", {
+              yearly_price: props.yearlyBilled.yearly_price,
+            })}
+          </small>
+          <span className={styles.price}>
+            {l10n.getString("plan-matrix-price-monthly-calculated", {
+              monthly_price: props.yearlyBilled.monthly_price,
+            })}
+          </span>
+        </div>
+        <a
+          ref={yearlyButtonRef}
+          href={props.yearlyBilled.subscribeLink}
+          onClick={() =>
+            trackPlanPurchaseStart(gaEvent, props.yearlyBilled.plan, {
+              label: props.yearlyBilled.gaViewPing?.label,
+            })
+          }
+          tabIndex={0}
+          className={styles["pick-button"]}
+        >
+          {l10n.getString("plan-grid-card-btn")}
+        </a>
+      </Item>
+      <Item
+        key="monthly"
+        title={l10n.getString("plan-matrix-price-period-monthly")}
+      >
+        <div className={styles["price-text"]}>
+          <small>{l10n.getString("plan-grid-billed-monthly")}</small>
+          <span className={styles.price}>
+            {l10n.getString("plan-matrix-price-monthly-calculated", {
+              monthly_price: props.monthlyBilled.monthly_price,
+            })}
+          </span>
+        </div>
+        <a
+          ref={monthlyButtonRef}
+          href={props.monthlyBilled.subscribeLink}
+          onClick={() =>
+            trackPlanPurchaseStart(gaEvent, props.monthlyBilled.plan, {
+              label: props.monthlyBilled.gaViewPing?.label,
+            })
+          }
+          tabIndex={0}
+          className={styles["pick-button"]}
+        >
+          {l10n.getString("plan-grid-card-btn")}
+        </a>
+      </Item>
+    </PricingTabs>
+  );
+};
+
+const PricingTabs = (props: TabListProps<object>) => {
+  const tabListState = useTabListState(props);
+  const tabListRef = useRef(null);
```

Comment:
question (non-blocking): are we using these refs for anything? Are they necessary?